docs/howto/system_testing.md

```diff
@@ -106,6 +107,55 @@ volumes:
   mysqldata:
 ```
 
+### Agent service deployer
+
+When using the Agent service deployer, the `elastic-agent` provided by the stack
+will not be used. An agent will be deployed as a Docker compose service named `docker-custom-agent`
+which base configuration is provided [here](../../internal/testrunner/runners/system/servicedeployer/custom-agent-base-config.yml).
+This configuration will be merged with the one provided in the `custom-agent.yml` file.
+This is useful if you need different capabilities than the provided by the
+`elastic-agent` used by the `elastic-package stack` command. 
+
+`custom-agent.yml`
```

Comment:
As the `custom-agent.yml` will be part of a package, we need to cover it with package-spec. You will need to open one more PR.